src/CALPHADConcSolverBinaryThreePhase.cc

```diff
@@ -3,6 +3,8 @@
 #include "xlogx.h"
 #include <iostream>
 
+#define EQSCALING 1.e-4
```

Comment:
A comment on why this particular value was chosen could be helpful. Is this based on some dimensional scaling? Or is it a empirically chosen value that improves convergence?

Reply:
What I really wanted is a scaling that is independent of T. 1.e-4 is close to what we had before with RT=8.31*1000 for T=1000 K and is now independent of T